news/2024.html

```diff
@@ -30,6 +30,34 @@
 </div>
 -->
 
+<div class="news-item tag-link" id="aarch64-fc">
+  <a href="#aarch64-fc"><span class="tag-span"># </span></a> 19 Apr
+  2024: Functional correctness proved for AArch64
+</div>
+<div>
+ <p>
+
+   <a href="https://www.rtx.com/">
+     <img src="/images/Proof-FC.svg"
+	  style="width: 20%;  padding-left:10px;  float:right"
+	  alt="Functional correctness proof"/>
+   </a>
+
+   We are extremely pleased to announce that the functional
+   correctness proof for seL4 on the 64-bit Arm architecture
+   (AArch64) is complete!<br/>
```

Comment:
I would suggest adding a link to https://proofcraft.systems/news-2024/#2024-04-01

Reply:
done